rotkehlchen/tests/utils/dataimport.py

```diff
@@ -1925,8 +1925,13 @@ def assert_binance_import_results(rotki: Rotkehlchen):
     assert asset_movements == expected_asset_movements
     assert expected_events == events
     expected_warnings = [
-        '2 Binance rows have bad format. Check logs for details.',
-        'Skipped 4 rows during processing binance csv file. Check logs for details',
+        'Failed to deserialize a timestamp from a null entry in binance',
+        'Unknown asset  provided.',
```

Comment:
I've updated it here: https://github.com/nicholasyoder/rotki/blob/csv-import/rotkehlchen/data_import/importers/binance.py#L691 to include double quotes around the asset, but this specific message is no longer present in the test because there was actually a mistake in the importer logic that I fixed and that entry is now being imported without error (See https://discord.com/channels/657906918408585217/1080783409640648765/1278772451634970725).

It is quite possible that this error could still happen though, so I think adding the double quotes is still a good idea.